pkg/lint/lintersdb/validator.go

```diff
@@ -55,56 +75,3 @@ func (v Validator) validatePresets(cfg *config.Linters) error {
 
 	return nil
 }
-
```

Comment:
by this logic `if len(cfg.Presets) != 0 && cfg.EnableAll {` should be moved in `Config.Validate` too?

Reply:
We can because of cyclic dependencies, I tried several possibilities but all go to the same cyclic dependencies problem.